go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/adamthesax/grpc-proxy v0.0.0-20220525203857-13e92d14f87a
 	github.com/armon/go-metrics v0.4.1
 	github.com/hashicorp/consul-server-connection-manager v0.1.0
-	github.com/hashicorp/consul/proto-public v0.1.1
+	github.com/hashicorp/consul/proto-public v0.1.2-0.20230111184009-6d2880e89478
```

Comment:
Not sure what we want to do here:
1. Merge and change this as part of the release
1. Leave it unmerged and fix right after the Consul Core release.

Reply:
Made a PR to update the release process in our docs IAW option 1. 👍